Sources/PackageDescription/Order.swift

```diff
@@ -0,0 +1,8 @@
+/// Order when the action gets executed.
+///
+/// - pre: Before commiting and tagging new version.
+/// - post: After commiting and tagging new version.
+public enum Order: String, Codable {
+    case pre
```

Comment:
❕ I think that also cases should be documented, not just mentioned in docs of the `enum`